server/clearbit.js

```diff
@@ -154,32 +164,32 @@ export default class Clearbit {
     const domain = getDomain(user);
 
     if (!this.client || !discover_enabled || _.isEmpty(discover_segments)) {
-      this.debug("Discover not enabled", { discover_segments });
+      this.logSkip(user, "discover", "Discover not enabled", { discover_segments });
```

Comment:
I think both ideas are not contradictory : 

Set the client scopped to the user at the beginning of the function
`const as_user = hull.asUser()`
And then call skip to factorize logs
`this.skip(as_user, "No domain", action: "prospector")`

With the skipped function being
```js
logSkip = (as_user, action, reason, additionalData = {}) => {
  as_user.logger.info("outgoing.user.skip", { reason, action, additionalData });
};
```